src/components/InputPasswordToggle/InputPasswordToggle.tsx

```diff
@@ -1,9 +1,17 @@
-import React, { Component } from 'react';
+import * as React from 'react';
 import classnames from 'classnames';
-import EyeSVG from '../../svg/eye.svg';
+import EyeSVG from '../../svg/eye';
+import IReactComponentProps from '../../common/structures/IReactComponentProps';
 
-export default class InputPasswordToggle extends Component {
-	constructor (props) {
+interface IState {
+
```

Comment:
I've seen this spacing on the interfaces quite a bit. Do you prefer the padding?

Reply:
My thinking was to be consistent with the spacing for both interfaces and classes. I'm okay with or without them, but my preference is to apply the same treatment to both.